manifests/image-references

```diff
@@ -78,3 +78,7 @@ spec:
     from:
       kind: DockerImage
       name: quay.io/openshift/origin-cli:latest
+  - name: metallb-frr
```

Comment:
```suggestion
  - name: metallb-frr-rhel9
```
To be consistent with https://github.com/openshift-eng/ocp-build-data/blob/2c5fb910532b1e3e418842d141eb45fd51dd2a83/images/ose-frr.yml#L24 and

https://github.com/openshift/metallb-operator/blob/6de2375a6350e62c0e34a9ebdd392b0832b76b86/manifests/stable/image-references#L14